src/theme/css/general.css

```diff
@@ -204,6 +204,9 @@ svg {
 #sidebar a.active {
 	color: var(--sidebar-active);
 }
+#sidebar a.active::after {
+	content: ' •' / ' selected';
+}
```

Comment:
how about this?

```suggestion
#sidebar a.active::after {
 	--size: 12px;
	content: "" / "selected";
	display: inline-block;
	width: var(--size);
	height: var(--size);
	-moz-border-radius: calc(var(--size) / 2);
	-webkit-border-radius: calc(var(--size) / 2);
	border-radius: calc(var(--size) / 2);
	background: var(--sidebar-active);
	margin-left: 0.5em;
}
```
- a bit bigger
- won't get messed up by font changes
- more centered

![image](https://github.com/user-attachments/assets/de99964f-7907-4cd6-bde3-0f1fdf5e2c9a)